Umbraco-StarterKit/Going-live/index.md

```diff
@@ -4,4 +4,64 @@ versionFrom: 8.0.0
 
 # Going live
 
-In this section you can find all the information needed in order to publish your website to the world wide web.
+In this section, you can find all the information needed in order to publish your website to the world wide web.
+
+## Domain/Hostname
+
+### Configure DNS records
+
+The first step is to set up your Hostname, you can do this from the portal.
+This guide will show you how to bind your hostname to your Umbraco Uno environment where you see your environment.
+
+It is recommended to set-up an ALIAS record for your root hostname (e.g. mysite.s1.umbraco.io), rather than an A record for the umbraco.io service IP address - 23.100.15.180.
+Check with your DNS host or hostname provider for details on how to configure this for your hostnames.
+
+### Add Hostname from the portal
+
+- Click Settings
+- Select Hostnames
+- Once inside Hostnames, click "Add new hostname"
+- Add your hostname to the field and select add
+- To keep track of the process, you can refresh the page when it is done sucessfully it will say: Protected by LATCH  
+
+### Umbraco LATCH
+
+Every Umbraco Project is protected by LATCH witch means that a TLS Certificate will be applied automatically.
+If you are interested in learning more about [LATCH](https://umbraco.com/products/umbraco-cloud/umbraco-latch/) this is a good place to start.
+
+### SMTP
+
+In Umbraco Uno projects we take care of SMTP(Simple Mail Transfer Protocol). So that you dont have to do anything.
+
+## SEO
+
+Setting up should be part of your general workflow for each of your nodes. This is to make your page the most visible in search engines as possible, all of the fields below should be filled out.
+
+- Company Name
+- Site Name
+- Company Logo
+- Phone Number
+- Email
+- Latitude
+- Longitude
+
+You can reach the general SEO Under your home node > Settings > General.
+
+There are more documentation about SEO at these pages [General settings](../Uno-pedia/Settings/General-Settings/index.md/#SEO) and [Specific settings ](../Uno-pedia/Settings/Specific-Settings/index.md/#SEO) 
+
+## Newsletter
```

Comment:
I think this needs a bit more info about the Newsletter widget, and perhaps also a link to it?